/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:33971'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:43636'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:34876'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:34082'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:38904'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:41041'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:34345'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:34824'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:45018'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:43475'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:46392'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:35979'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:40713'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:45376'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:44876'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:41239'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:45181'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:42805'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:37324'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:40163'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:42083'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:34471'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:45595'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:43170'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:37588'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:34443'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:45059'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.9.104:39959'
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:41873
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:41873
distributed.worker - INFO -          dashboard at:         172.30.9.104:38752
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:39375
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:44587
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:44698
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:44028
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:35444
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:39375
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:44302
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:44587
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:44028
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:35444
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:44698
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -          dashboard at:         172.30.9.104:45809
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:44302
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:46267
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:43965
distributed.worker - INFO -          dashboard at:         172.30.9.104:42711
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-817obwww
distributed.worker - INFO -          dashboard at:         172.30.9.104:46700
distributed.worker - INFO -          dashboard at:         172.30.9.104:37162
distributed.worker - INFO -          dashboard at:         172.30.9.104:32982
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO -          dashboard at:         172.30.9.104:35916
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:46267
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:43965
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          dashboard at:         172.30.9.104:42398
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          dashboard at:         172.30.9.104:39139
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -               Threads:                          1
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:36405
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-yponueq6
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-7k9argic
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:36405
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-r00voap7
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-r301bd5e
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-rlql1ve_
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-muy39oi5
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-mozb3iub
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -          dashboard at:         172.30.9.104:42654
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-sv297eq5
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-r8xrrfnt
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:40098
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:40098
distributed.worker - INFO -          dashboard at:         172.30.9.104:34914
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:35759
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:35759
distributed.worker - INFO -          dashboard at:         172.30.9.104:36123
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-b4vg6y5x
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-t38u60df
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:33680
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:33680
distributed.worker - INFO -          dashboard at:         172.30.9.104:38004
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-89215csm
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:34697
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:41444
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:34697
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:41444
distributed.worker - INFO -          dashboard at:         172.30.9.104:41504
distributed.worker - INFO -          dashboard at:         172.30.9.104:43957
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-5rg6dkp4
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-vksb1mjd
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:35316
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:45885
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:35316
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:45885
distributed.worker - INFO -          dashboard at:         172.30.9.104:45538
distributed.worker - INFO -          dashboard at:         172.30.9.104:38860
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:33978
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:33978
distributed.worker - INFO -          dashboard at:         172.30.9.104:41485
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-s67j2lyl
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-eacf5dcw
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-a510z29n
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:43764
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:43764
distributed.worker - INFO -          dashboard at:         172.30.9.104:46003
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:43144
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:43144
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -          dashboard at:         172.30.9.104:42620
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-6hs03rhx
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-q2aa8tp7
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:42641
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:42641
distributed.worker - INFO -          dashboard at:         172.30.9.104:36862
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-b9cf8hb0
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:40379
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:40379
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:40761
distributed.worker - INFO -          dashboard at:         172.30.9.104:36805
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:40761
distributed.worker - INFO -          dashboard at:         172.30.9.104:43698
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -                Memory:                    4.29 GB
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:34831
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-lizrkfle
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-5di0wu6x
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:34831
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          dashboard at:         172.30.9.104:40256
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-6y7vx7h6
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:36861
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:41083
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:36861
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:41083
distributed.worker - INFO -          dashboard at:         172.30.9.104:42278
distributed.worker - INFO -          dashboard at:         172.30.9.104:42775
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -               Threads:                          1
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:32968
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:32968
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-o5vuadqf
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-ukjq6_pn
distributed.worker - INFO -          dashboard at:         172.30.9.104:37993
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-igrl98zc
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:46485
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:46485
distributed.worker - INFO -          dashboard at:         172.30.9.104:33747
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-ffr533gs
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.core - INFO - Starting established connection
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/multiprocessing/semaphore_tracker.py:144: UserWarning: semaphore_tracker: There appear to be 1 leaked semaphores to clean up at shutdown
  len(cache))
distributed.nanny - WARNING - Worker process 56590 was killed by signal 7
distributed.nanny - WARNING - Restarting worker
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.9.104:44046
distributed.worker - INFO -          Listening to:   tcp://172.30.9.104:44046
distributed.worker - INFO -          dashboard at:         172.30.9.104:34720
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/worker-6_wbnsaj
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.4:46543
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
slurmstepd: error: *** JOB 7304591 ON n2799 CANCELLED AT 2019-08-28T12:10:17 DUE TO TIME LIMIT ***
